openedx/features/ucsd_support/tests/tests.py

```diff
@@ -0,0 +1,55 @@
+import json
+import mock
+
+from django.test import TestCase
+from django.urls import reverse
+from django.test import Client
+
+
+class TestEmailSendFromSupport(TestCase):
```

Comment:
Please rename this class as `SupportEmailTestCase`

Reply:
done